Traceback (most recent call last):
  File "c:\users\dulla\pycharmprojects\thesis_data_analisys\venv\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "c:\users\dulla\pycharmprojects\thesis_data_analisys\venv\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "c:\users\dulla\pycharmprojects\thesis_data_analisys\venv\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "c:\users\dulla\pycharmprojects\thesis_data_analisys\venv\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\DULLA\AppData\Local\Programs\Python\Python38\lib\asyncio\base_events.py", line 608, in run_until_complete
    return future.result()
  File "c:\users\dulla\pycharmprojects\thesis_data_analisys\venv\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "c:\users\dulla\pycharmprojects\thesis_data_analisys\venv\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "c:\users\dulla\pycharmprojects\thesis_data_analisys\venv\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
G = nx.Graph()
G = nx.Graph()

for i in range(0, 2):
    G.add_node(i)

G.add_edge(0, 1)
G.add_edge(0, 2)
nx.draw(G, with_labels=True, font_weight='bold', node_size=1000, node_color='black', font_color='white')
plt.show()
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mNameError[0m                                 Traceback (most recent call last)
Input [1;32mIn [2][0m, in [0;36m<cell line: 1>[1;34m()[0m
[1;32m----> 1[0m G [38;5;241m=[39m [43mnx[49m[38;5;241m.[39mGraph()
[0;32m      2[0m G [38;5;241m=[39m nx[38;5;241m.[39mGraph()
[0;32m      4[0m [38;5;28;01mfor[39;00m i [38;5;129;01min[39;00m [38;5;28mrange[39m([38;5;241m0[39m, [38;5;241m2[39m):

[1;31mNameError[0m: name 'nx' is not defined
NameError: name 'nx' is not defined

